src/BulletDynamics/Dynamics/btDiscreteDynamicsWorld.cpp

```diff
@@ -1175,12 +1175,12 @@ void btDiscreteDynamicsWorld::debugDrawConstraint(btTypedConstraint* constraint)
 				//const btScalar length = btScalar(5);
 				const btScalar length = dbgDrawSize;
 				static int nSegments = 8 * 4;
-				btScalar fAngleInRadians = btScalar(2. * 3.1415926) * (btScalar)(nSegments - 1) / btScalar(nSegments);
+				btScalar fAngleInRadians = btScalar(2. * 3.14159265) * (btScalar)(nSegments - 1) / btScalar(nSegments);
 				btVector3 pPrev = pCT->GetPointForAngle(fAngleInRadians, length);
 				pPrev = tr * pPrev;
 				for (int i = 0; i < nSegments; i++)
 				{
-					fAngleInRadians = btScalar(2. * 3.1415926) * (btScalar)i / btScalar(nSegments);
+					fAngleInRadians = btScalar(2. * 3.14159265) * (btScalar)i / btScalar(nSegments);
```

Comment:
Considering `btScalar` can be a `double` if compiled with `BT_USE_DOUBLE_PRECISION=1`, shouldn't this be even more precise?

```suggestion
					fAngleInRadians = btScalar(2. * 3.1415926535897932) * (btScalar)i / btScalar(nSegments);
```

Or maybe:

```suggestion
					fAngleInRadians = btScalar(6.2831853071795864) * (btScalar)i / btScalar(nSegments);
```